reveal/run.js

```diff
@@ -88,8 +91,23 @@
     });
   }
 
-  function hasRemoteQueryParameter() {
-    return window.location.search.match(/[?&]remote[&]?/i);
+  function selectWidth(queryParams) {
+    if (queryParameterIsPresent(window.location.search, 'wide')) return 1413; // 16:9
+    else if (queryParameterIsPresent(window.location.search, 'wide10')) return 1272; // 16:10
+    else return 1124; // ~4:3
+  }
+
+  function cycleWideModes() {
+    var queryString = window.location.search;
```

Comment:
What if we need to remove a parameter and add two? We would need yet another function. I wanted to prevent this combinatorial explosion.